src/mlpack/core/data/image_info_impl.hpp

```diff
@@ -13,6 +13,24 @@
 #ifndef MLPACK_CORE_DATA_IMAGE_INFO_IMPL_HPP
 #define MLPACK_CORE_DATA_IMAGE_INFO_IMPL_HPP
 
+namespace mlpack {
+namespace data {
+
+inline const std::vector<std::string> LoadFileTypes()
+{
+  return std::vector<std::string>({"jpg", "png", "tga",
+    "bmp", "psd", "gif", "hdr", "pic", "pnm", "jpeg"});
+}
+
+inline const std::vector<std::string> SaveFileTypes()
+{
+  return std::vector<std::string>({"jpg", "png", "tga",
+    "bmp", "hdr"});
```

Comment:
```suggestion
  return std::vector<std::string>({"jpg", "png", "tga", "bmp", "hdr"});
```
This one fits all on one line. :+1: